shell/platform/cpp/BUILD.gn

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("$flutter_root/common/config.gni")
```

Comment:
The convention followed is `//flutter/shell/platform/<name_of_platform>`. Since `cpp` is not a platform, this location does not make sense. Please move this to `platform/linux`. If this is going to be a thing on Windows too, `platform/glfw` with its own subdirectory for linux and Windows specific files (if any).